BraveShared/Preferences.swift

```diff
@@ -29,7 +29,17 @@ extension Preferences {
     final class DAU {
         static let lastLaunchInfo = Option<[Int?]?>(key: "dau.last-launch-info", default: nil)
         static let weekOfInstallation = Option<String?>(key: "dau.week-of-installation", default: nil)
-        static let firstPingParam = Option<Bool>(key: "dau.first-ping", default: true)
+        static let firstPingParam: Option<Bool> = {
```

Comment:
We can literally just set the `defaultValue` inline:

(key: "", default: Preferences.DAU.lastLaunchInfo.value == nil`)

This then would set `default` to the correct value.